ts/WoltLabSuite/Core/Ui/Search/Extended.ts

```diff
@@ -167,7 +189,15 @@ export class UiSearchExtended {
     const data = {};
     new FormData(this.form).forEach((value, key) => {
       if (value.toString()) {
-        data[key] = value;
+        if (data[key] === undefined) {
+          data[key] = value;
+        } else {
+          if (!Array.isArray(data[key])) {
+            data[key] = [data[key]];
+          }
+
+          (data[key] as Array<any>).push(value);
```

Comment:
This likely does not do the correct thing:

If multiple items are selected, an array will be sent, but if only a single item is selected, a string will be sent. It should probably always be a (single-element) array if the element supports multiple selections.

In any case the amount of typechecks required here are not a good style, but this will likely solve itself with the previous paragraph.

Reply:
The problem is to determine the exact case when it can/should be an array/multi-value property. The current logic (`FormData`) does not support getting the HTML-elements in order to determine the `multiple`-attribute, so we'd have to re-integrate the whole logic to force an array for those properties.